test/e2e/tests/import-tokens.spec.js

```diff
@@ -0,0 +1,61 @@
+const { strict: assert } = require('assert');
+const {
+  withFixtures,
+  convertToHexValue,
+  regularDelayMs,
+} = require('../helpers');
+const FixtureBuilder = require('../fixture-builder');
+
+describe('Import flow', function () {
+  const ganacheOptions = {
+    accounts: [
+      {
+        secretKey:
+          '0x7C9529A67102755B7E6102D6D950AC5D5863C98713805CEC576B945B15B71EAC',
+        balance: convertToHexValue(25000000000000000000),
+      },
+    ],
+  };
+
+  it('allows importing multiple tokens from search', async function () {
+    await withFixtures(
+      {
+        fixtures: new FixtureBuilder().build(),
+        ganacheOptions,
+        title: this.test.title,
+      },
+      async ({ driver }) => {
+        await driver.navigate();
+        await driver.fill('#password', 'correct horse battery staple');
+        await driver.press('#password', driver.Key.ENTER);
+
+        // Token list is only on mainnet
+        await driver.clickElement('[data-testid="network-display"]');
+        await driver.clickElement({ text: 'Ethereum Mainnet', tag: 'button' });
+
+        // Wait for network to change and token list to load from state
+        await driver.delay(regularDelayMs);
```

Comment:
question: there's a delay in finding elements `[data-testid="import-token-button"]` when we execute `clickElement`, so is this still necessary?

Reply:
Having arbitrary delays is a bad thing but when I do something like the following:

```
await driver.waitForElementNotPresent('.loading-overlay');
```

...the detection is so fast that the search list isn't available yet, and so a search shows no results.